### DISCUSSION TITLE
Cannot unmarshal common settings - Invalid character

### DISCUSSION THREAD

**Discussion by bmpreston** [qdm12/ddns-updater#476] [General]
New install and got passed an issue with the data folder.  My issue is an invalid character, but can't seem to find where this double slash is called on.  

```
2023-05-22T20:11:27-04:00 INFO reading JSON config from file /updater/data/config.json
2023-05-22T20:11:27-04:00 ERROR cannot unmarshal common settings: invalid character '\\' looking for beginning of object key string
2023-05-22T20:11:27-04:00 INFO Shutdown successful
```

Config.json:

```json
{
  "settings": [
    {
      "provider": "google",
      "domain": “*********”,
      "host": "@",
      "username": “*********”,
      "password": “*************”,
      "ip_version": "ipv4"
    }
  ]
}
```

**Comment by qdm12** [comment]
Maybe your domain, username and password double quotes `”` should be replaced with `"`?
If you literally copied pasted your json config, highlighting it with json on your original comment shows those quotes in red, maybe that's the cause?

Otherwise, do you have a `\\` somewhere? Are you sure the bind mounted file is the correct one?

**Reply by bmpreston** [reply]
Seems I found the problem.  When I re-downloaded the config file, it was clobbered.  I don't know what happened to it, but it was edited with XML markup.  I re-wrote it, uploaded, logs say things are fine now.  

Thanks!
